### DISCUSSION TITLE
请问 2.x 中如何实现 小地图中用色块代表节点，隐藏了原始节点的细节。

### DISCUSSION THREAD

**Discussion by JoeABCDEF** [antvis/X6#3384] [Q&A]
![image](https://user-images.githubusercontent.com/39638510/225278575-23b887ce-4148-4041-ad0a-64b3b6cbefb4.png)

我找到了实现 隐藏线的 方法, 但是 节点 用色块代替 测试一直不生效, 使用 new CellView 直接不显示了, 不知道咋实现, 请问可以出个 demo吗

**Comment by JoeABCDEF** [comment]
有没有大佬知道咋用吗, 可以给个代码吗

**Reply by NewByVector** [reply]
使用 [createCellView](http://x6.antv.antgroup.com/api/graph/view#createcellview) 配置。

**Reply by JoeABCDEF** [reply]
> 使用 [createCellView](http://x6.antv.antgroup.com/api/graph/view#createcellview) 配置。

@NewByVector  知道是createCellView, 边不显示返回null,  但是 节点 不知道咋返回, 试了 new CellView()  显示空白不知道是因为啥

**Reply by NewByVector** [reply]
参考这个[代码](https://x6.antv.vision/zh/docs/tutorial/basic/minimap#%E6%BC%94%E7%A4%BA)。

**Reply by JoeABCDEF** [reply]
> 代码

@NewByVector  但是 2.x 好像 没这个 方法了getCellView
然后下边 尝试返回 SimpleNodeView, 返回 CellView 啥的都不生效 直接就不渲染, 然后就不清楚 在 用什么方法了

<img width="519" alt="image" src="https://user-images.githubusercontent.com/39638510/227097814-aafe512c-1b9a-41ee-8f40-c4232a74831b.png">

**Reply by zhangyucha0** [reply]
同问

**Reply by JoeABCDEF** [reply]
> 同问
@zhangyucha0 

[minimap](https://x6.antv.antgroup.com/tutorial/plugins/minimap);
这里有, 但是需要升级minimap 可以看示例中的 版本
